### PR TITLE
dlg's tada68 keymap

### PR DESCRIPTION
**Pull request by dlgoodr**
[tshack's layout](https://github.com/qmk/qmk_firmware/tree/master/keyboards/tada68/keymaps/tshack), with:
 + Mac layer
   + Press Fn-I to align the bottom row to mac standards
 + volume controls moved to Fn+ M<>
 + [Space Cadet Shift](https://docs.qmk.fm/#/feature_space_cadet_shift) enabled
 + Fn+R for LED breathing
 + Fn+Del for Insert
 + `GRAVE_ESC_ALT_OVERRIDE` so cmd-opt-esc still works (thx [mattdicarlo](https://github.com/qmk/qmk_firmware/tree/master/keyboards/tada68/keymaps/mattdicarlo))

### CODE REVIEW

File: keyboards/tada68/keymaps/dlg/keymap.c [hunk 1]
```diff
@@ -0,0 +1,78 @@
+#include QMK_KEYBOARD_H
+
+#define _BL 0
+#define _FL 1
+#define _MAC 2
+
+#define _______ KC_TRNS
```
Comment (mechmerlin) [inline keyboards/tada68/keymaps/dlg/keymap.c]: 7 space (_______) is already defined to be KC_TRNS in `QMK_KEYBOARD_H`. You don't need this line

```suggestion

```

File: keyboards/tada68/keymaps/dlg/rules.mk [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+NKRO_ENABLE = yes           # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight. 
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+
+ifndef QUANTUM_DIR
```
Comment (mechmerlin) [inline keyboards/tada68/keymaps/dlg/rules.mk]: You can remove lines 19-21. They are not needed here.

```suggestion

```

File: keyboards/tada68/keymaps/dlg/config.h [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#include "../../config.h"
```
Comment (noroadsleft) [inline keyboards/tada68/keymaps/dlg/config.h]: This line should be changed to:

```suggestion
#pragma once
```

The build script will include the referenced file automatically without this line, and we only want this file to be processed once.

**Comment by noroadsleft** [comment]
A suggestion, but it doesn't pertain to your submitted files.

I noticed that you're filing this pull request from your fork's `master` branch. While this isn't a major issue for this PR, later changes may result in merge conflicts if you try to file another PR because the commit history will have diverged between your master and QMK's. It is highly recommended for QMK development - regardless of what is being done or where - to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.

There are instructions on how to keep your master updated here:

[**Best Practices: Your fork's master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_best_practices?id=your-fork39s-master-update-often-commit-never)

If you need any help with this just ask.

**Comment by dlgoodr** [comment]
Thanks for the coaching.  I know better, I deliver the message all the time at work.  I got lazy :)

**Comment by dlgoodr** [comment]
Now what?

**Comment by drashna** [comment]
Thanks!

**Pull request merged**